src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -249,6 +253,19 @@ public Translation2d getTranslationTarget() {
                 return translationTarget;
         }
 
+        public void setBalanceTarget(Rotation2d balanceTarget) {
+                this.balanceTarget = balanceTarget;
+                drive(0, 0, 0);
```

Comment:
We should capture the current pose or reset the translatedMeters variable when the target is set.